src/components/details-card/details-card.tsx

```diff
@@ -53,33 +56,40 @@ const DetailsCard: React.FC<any> = (props) => {
   );
   const [secret, secretloaded, secretloadError] =
     useK8sWatchResource<SecretKind>(flashSecretResource);
+  const secretData = getEndpoint(secret);
   const endpointAddress = React.useMemo(
     () =>
-      secretloaded && !secretloadError
-        ? Base64.decode(getEndpoint(secret))
-        : "",
+      secretloaded && !secretloadError && secretData
+        ? Base64.decode(secretData)
+        : "unknown",
     [data, secretloaded, secretloadError]
   );
 
-  const flashOperatorVersion = getIBMStorageODFVersion(subscriptions);
-  const operatorPath = `${resourcePathFromModel(
-    ClusterServiceVersionModel,
-    flashOperatorVersion,
-    stoData?.metadata.namespace
-  )}`;
+  const flashOperatorVersion =
+    flashsystemloaded && !flashsystemloadError
+      ? getIBMStorageODFVersion(subscriptions)
+      : "unknown";
+  const operatorPath =
+    subscriptionloaded && !subscriptionloadError
+      ? `${resourcePathFromModel(
+          ClusterServiceVersionModel,
+          flashOperatorVersion,
+          stoData?.metadata.namespace
+        )}`
+      : "unknown";
```

Comment:
This will who then "https:unknown " ?
I think you would rather want to say "Not available"
```suggestion
      : null;
```